projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-util.ts

```diff
@@ -212,8 +212,13 @@ export class PivotUtil {
         data.forEach(rec => {
             const children = rec.children;
             if (children) {
-                children.forEach(x => {
-                    this.sort(x, expressions, sorting);
+                children.forEach((value, key, map) => {
```

Comment:
I would migrate the `forEach` invocation(s) to a `for`, or a `for..of` loop. It's a nitpick but they usually have better performance characteristics for bigger datasets.